roles/influxdb2/tasks/install.yml

```diff
@@ -6,10 +6,26 @@
       - python3-pip
     state: present
 
-- name: Install influxb-client for python
+- name: Install influxdb-client via block for rescue
+  when:
+    - not influxdb_force_pip | bool
+  block:
+    - name: Install influxdb-client for python
```

Comment:
Currently the full "path" of the modules are used inside the roles. Please use it here also. `ansible.builtin.XXX`

Reply:
Changed it. (Never used full path with `block`, but changed `fail`.

I wonder, why `ansible-lint` didn't catch that.

Feel free to close the conversation.